docs/core/extensions/logging.md

```diff
@@ -87,29 +87,75 @@ In the preceding sample:
 
 ### Set log level by command line, environment variables, and other configuration
 
-Log level can be set by any of the [configuration providers](configuration-providers.md).
+Log level can be set by any of the [configuration providers](configuration-providers.md). For example, you can create a persisted environment variable named `Logging:LogLevel:Microsoft` with a value of `Information`.
 
-The following commands:
+## [Command Line](#tab/command-line)
 
-- Set the environment key `Logging:LogLevel:Microsoft` to a value of `Information` on Windows.
-- Test the settings when using an app created with the .NET Worker service templates. The `dotnet run` command must be run in the project directory after using `set`.
+Create and assign persisted environment variable, given the log level value.
 
-```cmd
-set Logging__LogLevel__Microsoft=Information
-dotnet run
+```CMD
+:: Assigns the env var to the value
+setx "Logging__LogLevel__Microsoft" "Information" /M
+```
+
+In a *new* instance of the **Command Prompt**, read the environment variable.
+
+```CMD
+:: Prints the env var value
+echo %Logging__LogLevel__Microsoft%
+```
+
+## [PowerShell](#tab/powershell)
+
+Create and assign persisted environment variable, given the log level value.
+
+```powershell
+# Assigns the env var to the value
+[System.Environment]::SetEnvironmentVariable(
+    "Logging__LogLevel__Microsoft", "Information", "Machine")
+```
+
+In a *new* instance of the **PowerShell**, read the environment variable.
+
+```powershell
+# Prints the env var value
+[System.Environment]::GetEnvironmentVariable(
+    "Logging__LogLevel__Microsoft", "Machine")
 ```
 
-The preceding environment setting:
+## [Bash](#tab/bash)
 
-- Is only set in processes launched from the command window they were set in.
-- Isn't read by apps launched with Visual Studio.
+Create and assign persisted environment variable, given the log level value.
 
-The following [setx](/windows-server/administration/windows-commands/setx) command also sets the environment key and value on Windows. Unlike `set`, `setx` settings are persisted. The `/M` switch sets the variable in the system environment. If `/M` isn't used, a user environment variable is set.
+```Bash
+# Assigns the env var to the value, persists it across sessions
+echo export Logging__LogLevel__Microsoft="Information" >> ~/.bashrc && source ~/.bashrc
+```
+
+To read the environment variable.
+
+```Bash
+# Prints the env var value
+echo $Logging__LogLevel__Microsoft
 
-```cmd
-setx Logging__LogLevel__Microsoft=Information /M
+# Or use printenv:
+# printenv Logging__LogLevel__Microsoft
 ```
 
+> [!NOTE]
+> When configuring environment variables with names that contain periods `.`, consider the "Exporting a variable with a dot (.) in it" question on **Stack Exchange** and its corresponding [accepted answer](https://unix.stackexchange.com/a/93533).
```

Comment:
> When configuring environment variables with names that contain periods `.`,

I've been writing these as

When configuring environment variables with names that contain `.` (periods ),
because it seems to look better MT. It works everywhere to do the literal first. That's not standard practice however.

Reply:
I like that, thanks for calling attention to it. For future reference, feel free to make these an official suggestion - it's much easier to implement this feedback when it is a suggestion from the GitHub UI.